readme.txt

```diff
@@ -53,17 +51,17 @@ You can get started with ImageEngine easily with a [free, no credit card require
 * **Standard** - $99/month. Up to **250 GB** per month. Includes 3 custom domains (CNAME) with HTTPS support. Priority onboarding support.
 * **Pro** - pricing scales with usage volume.  WAF with DDoS protection. Dedicated edge servers available. Ticketed enterprise support. [Contact us](https://imageengine.io/contact?utm_source=wordpress.org&utm_medium=page&utm_term=wp-imageengine&utm_campaign=wp-imageengine).
 
-= Features =
+= Features & BENEFITS=
 
-* Maximize web performance by serving static assets from a CDN
+* Maximize web performance by automatically serving optimized WebP, JPEG 2000, or AVIF images  from a CDN
 * Set the WordPress directories that should be included
 * Custom filters "image_cdn_url" and "image_cdn_html" included
 * Define excluded directories or extensions
 * Enable or disable HTTPS support
 * Turn on or off quickly without deactivating the plugin
 * Test the CDN integration before saving your changes to make sure it will work properly
-* Supports [ImageEngine Directives](https://imageengine.io/docs/implementation/directives/?utm_source=wordpress.org&utm_medium=page&utm_term=wp-imageengine&utm_campaign=wp-imageengine)
-* Compatible with the [WordPress Cache Enabler](https://wordpress.org/plugins/cache-enabler/) plugin
+* Supports [ImageEngine Directives](https://imageengine.io/docs/implementation/directives/?utm_source=wordpress.org&utm_medium=page&utm_term=wp-imageengine&utm_campaign=wp-imageengine). Users of ImageEngine can also configure the Directives for their assets to control image quality, automatic format conversion and automatic image resizing.
+* Compatible with the [WordPress Cache Enabler](https://wordpress.org/plugins/cache-enabler/) plugin, WooCommerce, Elementor and other popular plugins.
```

Comment:
Maybe add a line for the page builders that are compatible (Gutenberg, Elementor, WPBakery / Visual Composer, Oxygen Builder, Divi) since these keywords are likely to be searched.